examples/todos/common/src/commonMain/kotlin/org/reduxkotlin/examples/todos/TodosReducer.kt

```diff
@@ -0,0 +1,15 @@
+package org.reduxkotlin.examples.todos
+
+
+fun todosReducer(state: List<Todo>, action: Any) =
+    when (action) {
+        is AddTodo -> state.plus(Todo(action.text))
```

Comment:
If you wanted to get real fancy, you could use operator overloading to do `state += Todo(action.next)`

Reply:
`state + Todo(action.next)` will work.  State is immutable here.  I like it